### ISSUE TITLE
Dependency Loading

### ISSUE THREAD

**Issue by Matt-MX** [Matt-MX/ReconnectVelocity#20]
Preferably we want to reduce the jar size since currently all drivers are shaded because Velocity does not provide a dependency loader system. If anyone has the time to make a PR or help implement this it'd be appreciated, noone wants to download a 23mb jar every update lol.

**Comment by StarGamerShalom** [comment]
Yeah I was about to ask why the jar went from 14 to 24 ouch. That is bigger than any plugin on the server including velocity or Geyser.
